frontend/src/ShareButton/ShareButton.module.css

```diff
@@ -0,0 +1,21 @@
+.share {
```

Comment:
⚠️ action required: oops, completely forgor to remove this. can this be merged with `Button`? that is, can the share icon just be added to that component?